SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSubviews.cpp

```diff
@@ -0,0 +1,219 @@
+#include "Interop/ActorSubviews.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/InitialOnlyFilter.h"
+#include "Schema/Restricted.h"
+#include "Schema/Tombstone.h"
+#include "Schema/UnrealMetadata.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/SubView.h"
+#include "SpatialView/ViewCoordinator.h"
+
+namespace SpatialGDK
+{
+namespace ActorSubviews
+{
+namespace MainActorSubviewSetup
+{
+static bool IsActorEntity(const Worker_EntityId EntityId, const EntityViewElement& Entity, USpatialNetDriver& NetDriver);
+
+static TArray<FDispatcherRefreshCallback> GetCallbacks(ViewCoordinator& Coordinator);
+} // namespace MainActorSubviewSetup
+
+namespace AuthoritySubviewSetup
+{
+static bool IsAuthorityActorEntity(const Worker_EntityId EntityId, const EntityViewElement& Element);
+
+static TArray<FDispatcherRefreshCallback> GetCallbacks(ViewCoordinator& Coordinator);
+} // namespace AuthoritySubviewSetup
+
+namespace AutonomousSubviewSetup
```

Comment:
I really wish we have cxx17 here, nested name spaces would make this much clearer.